i18n/events/lightning-talk.i18n.js

```diff
@@ -142,7 +142,7 @@ export default genI18nMessages({
                 title: '閃電秀規則',
                 description: '',
                 list: [
-                    '閃電秀{bold}主題{/bold}需與{bold}技術{/bold}、{bold}技術社群{/bold}相關，{bold}Python{/bold}相關主題尤其歡迎！',
+                    '閃電秀{bold}主題{/bold}需與{bold}技術{/bold}、{bold}技術社群{/bold}相關，{bold}Python{/bold} 相關主題尤其歡迎！',
                     '閃電秀的呈現內容{bold}必須遵守{/bold} {codeOfConduct}，避免造成聽眾不適的內容。',
```

Comment:
For the Chinese version, I was originally hoping to format it like this:
```
閃電秀的呈現內容{bold}必須遵守 {codeOfConduct}{/bold}
```

instead of:
```
閃電秀的呈現內容{bold}必須遵守{/bold} {codeOfConduct}
```

Not sure if it’s technically feasible though